CMake/utils/kwiver-utils-modules.cmake

```diff
@@ -23,7 +23,7 @@ define_property(GLOBAL PROPERTY kwiver_modules_enabled
 # enabled.
 #
 # The intended purpose for modules is ... (what is a module? How does
-# it differ from a library?)
+# it differ from a library?) Would this be like adding arrows? or arrows/core?
```

Comment:
This was something that came from sprokit. It was actually not used there , but does implement an interesting approach to managing "module" or component dependencies. As it stands now, I am not sure it provides enough value, but the approach could be adapted to manage the dependencies at cmake time. For example, if we have an arrow that requires OpenCV, caffe, and VXL, this could provide a way of checking to see if all required dependencies are enabled and issue helpful messages if not.